app/controllers/concerns/back_links.rb

```diff
@@ -21,34 +21,20 @@ def redirect_back_to_application_review_page?
 
   # Method to determine the path to the candidates current dashboard based on
   # contextual information.
+
   def application_form_path
     # `current_application` is a `helper_method` defined in CandidateInterfaceController
     # It's not available in view specs
     return '' unless defined?(current_application)
 
-    if current_application.continuous_applications?
-      continuous_application_form_path
-    else
-      old_cycle_application_form_path
-    end
-  end
-  module_function :application_form_path
-
-  def continuous_application_form_path
     if request.path.match?(/withdraw/)
       candidate_interface_continuous_applications_choices_path
     else
       candidate_interface_continuous_applications_details_path
```

Comment:
Thoughts on renaming `candidate_interface_continuous_applications_choices_path` to `candidate_interface_choices_path`. It's in 42 files.

and `candidate_interface_continuous_applications_details_path` to `candidate_interface_details_path`. It's in 28 files.

Reply:
We should definitely do it, but I'm happy for that to go in a separate card, since it has a wider impact than just the back links